File: metric_tool/test/resources/kotlin/rfc/RfcE.kt
package io.company.app.view.basket.kotlin.rfc

import io.company.app.view.basket.kotlin.rfc.other.RfcOther

internal class RfcE1 {
    fun method() {
        val rfcOther = RfcOther()
        rfcOther.method1()
        rfcOther.method2(1)
    }
}

internal class RfcE2 {
    fun method() {
        val rfcOther = RfcOther()
        rfcOther.method1()
    }
}

File: metric_tool/test/resources/dart/rfc/rfc_e.dart
library mylib;

import 'other/rfc_other.dart';

class RfcE1 {
  void method() {
    RfcOther rfcOther = RfcOther();
    rfcOther.method1();
    rfcOther.method2(1);
  }
}

class RfcE2 {
  void method() {
    RfcOther rfcOther = RfcOther();
    rfcOther.method1();
  }
}